### PR TITLE
Plans: Replace "priority support" feature items with one large clause

### PR DESCRIPTION
**Pull request by mattgawarecki**
#### Changes proposed in this Pull Request

Related to `1196108640073826-as-1199527862646257`.

* Remove "priority support" from all product and plan cards.
* Remove the `PRIORITY_SUPPORT_V2` feature object and slug, since they're no longer used.
* Add an asterisk clause to the middle of the product selector page, mentioning that all plans include priority support.

#### Testing instructions

* In any environment, open the Plans/Pricing page.
* Verify that product and plan cards no longer mention "priority support" in their list of features.
* Verify that a large asterisk clause is visible in the middle of the page stating "All plans include priority support" (see reference screenshot). Styling should match the mock-up at `zp0udhbEB9dvdS4rB3GJ0z-fi-923%3A0`.

#### Reference screenshot

image

### CODE REVIEW

File: client/my-sites/plans/jetpack-plans/i5/products-grid-i5/index.tsx [hunk 1]
```diff
@@ -182,6 +182,13 @@ const ProductsGridI5: React.FC< ProductsGridProps > = ( {
 					/>
 				</div>
 			</section>
+
+			<section className="products-grid-i5__section products-grid-i5__asterisk-items">
+				<h2 className="products-grid-i5__asterisk-item">
+					{ translate( 'All plans include priority support' ) }
```
Comment (a8ci18n) [inline client/my-sites/plans/jetpack-plans/i5/products-grid-i5/index.tsx:188]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'Personal Plans now Include Priority Support' )` *ES Score: 7*

**Review by rcanepa — APPROVED** [review]
LGTM! Code changes look fine and the UI looks as expected.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5517299

Thank you @mattgawarecki for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5517299).

**Pull request merged**